src/esthri/Cargo.toml

```diff
@@ -0,0 +1,121 @@
+[[bin]]
+name = "esthri"
+path = "src/main.rs"
+required-features = ["cli"]
+
+[package]
+name = "esthri"
+version = "4.1.3"
+authors = [ "Jason Mobarak <jason@swift-nav.com>",]
+edition = "2018"
+
+[dependencies]
+once_cell = "1.5"
+rust-crypto = "0.2"
+hex = "0.4"
+structopt = "0.3"
```

Comment:
Is there a way we can move the binary only dependencies to optional as well? It might make sense to break the cli and lib into two crates if we want to minimize time to build when consuming as a lib